docs/docs/guides/web3_eth_migration_guide.md

```diff
@@ -0,0 +1,1098 @@
+# Web3Eth Migration Guide
+
+## Breaking Changes
+
+### Not Implemented or Exported
+
+-   [extend](https://web3js.readthedocs.io/en/v1.7.3/web3-eth.html#extend) functionality not implemented
+-   [web3.eth.createAccessList](https://web3js.readthedocs.io/en/v1.7.3/web3-eth.html#createaccesslist) not implemented
+-   [web3.eth.personal](https://web3js.readthedocs.io/en/v1.7.3/web3-eth.html#personal) namespace is not exported
+-   [web3.eth.net](https://web3js.readthedocs.io/en/v1.7.3/web3-eth.html#net) namespace is not exported
+
+### Defaults and Configs
+
+-   `givenProvider` default value is `undefined` instead of `null`
+-   `currentProvider` will never return `null`, provider required upon instantiation as opposed to being optional in 1.x //todo maybe this is not true after after the recent discussion/change
+-   `web3.eth.defaultAccount` default value is `undefined` instead of `null`
+    -   1.x has `undefined` documented as default, but in implementation it's `null`
+-   `web3.eth.defaultHardfork` default is `london` instead of `undefined`
+    -   1.x has `london` documented as default, but in implementation it's `undefined`
+-   `web3.eth.defaultChain` default is `mainnet` instead of `undefined`
+    -   1.x has `mainnet` documented as default, but in implementation it's `undefined`
+
+### Web3Eth Methods
+
+### `web3.eth.getHashrate`
+
+renamed to `getHashRate`
+
+### `web3.eth.getGasPrice`
+
+returns a hex string instead of a number string
+
+```typescript
+// in 1.x
+await web3.eth.getGasPrice(); // '2000000000'
+
+// in 4.x
+await web3.eth.getGasPrice(); // '0x77359400'
+```
+
+### `web3.eth.getFeeHistory`
+
+returns hex strings for `gasUsedRation` array items instead of a numbers
```

Comment:
If it's defined as number, then it should be returning `bigint` not the hex string.